Formula/elvis.rb

```diff
@@ -0,0 +1,44 @@
+class Elvis < Formula
+  desc "Erlang Style Reviewer"
+  homepage "https://github.com/inaka/elvis"
+  url "https://github.com/inaka/elvis/archive/refs/tags/1.1.0.tar.gz"
+  sha256 "4afa3629f95997449ec9ce15a4aa59dd8fece5a5320e23e1d1c7590d1831d953"
+  license "Apache-2.0"
+  head "https://github.com/inaka/elvis.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Unless there are issues with the default livecheck this should be removed

Reply:
Aha, I didn't know there was a default one. Maybe something to add to the `brew create` template?

Reply:
The default one is to not have the `livecheck` at all, which is exactly what you get in the `brew create` template, no?

Reply:
I was wondering if there's a default implementation for programs hosted on GitHub releases?